src/types/inc/utils.hpp

```diff
@@ -52,6 +52,20 @@ namespace Microsoft::Console::Utils
     bool StringToUint(const std::wstring_view wstr, unsigned int& value);
     std::vector<std::wstring_view> SplitString(const std::wstring_view wstr, const wchar_t delimiter) noexcept;
 
+    enum PasteOption
+    {
+        // Convert Windows-space \r\n and \n line-endings to \r line-endings.

```

Comment:
Funny that this breaks the tooltip in VS:

![image](https://user-images.githubusercontent.com/4710575/106875762-01ab2e00-6712-11eb-87d0-1e2bffd0b341.png)